identifiers/country-es/comunidad_autonoma.csv

```diff
@@ -0,0 +1,19 @@
+id,name
```

Comment:
I think based on where the discussion is going in #170 we should use something like "community:" in place of "state:" here, but I'd like to get @jpmckinney's thoughts.

Reply:
aliases would also be possible?

ocd-division/country:es/community:ga/
"same as"
ocd-division/country:es/state:ga/

And yes, spain has in this case no states, but "autonomous communities".
What do you think?

Reply:
SGTM (minus the trailing `/`s, of course).

Reply:
Yes, without / at the end. My fault.

Reply:
I used now autonomous_community instead of state. And changes the code from hasc to iso fpr the autonomous_communitie's.
@jpmckinney @jdmgoogle 
Anything else here to do?